pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -182,6 +249,7 @@ func newKubeVirtForCR(cr *hcov1alpha1.HyperConverged) *kubevirt.KubeVirt {
 	labels := map[string]string{
 		"app": cr.Name,
 	}
+	log.Info("Creating a new KubeVirt object")
```

Comment:
this log should be on a different place. it implies (at least to me) that the object is being created on kube api, while you only initialize the structure here.